code/game/machinery/hologram.dm

```diff
@@ -41,313 +37,274 @@ var/const/HOLOPAD_MODE = RANGE_BASED
 	idle_power_usage = 5
 	use_power = 1
 
-	var/list/mob/living/silicon/ai/masters = new() //List of AIs that use the holopad
-	var/last_request = 0 //to prevent request spam. ~Carn
-	var/holo_range = 5 // Change to change how far the AI can move away from the holopad before deactivating
-	var/hacked = 0 // if hacked the hologram doesn't display. It only gives audio feedback.
-
-	var/incoming_connection = 0
-	var/mob/living/caller_id
-	var/obj/machinery/hologram/holopad/sourcepad
-	var/obj/machinery/hologram/holopad/targetpad
-	var/forced
-	var/last_message
+	var/holopad_id = ""
```

Comment:
```suggestion
	var/holopad_id
```

Being pedantic maybe, but I don't think this needs to be initialized. It should never be used without Initialize() first giving it a value from its area.